### ISSUE TITLE
[Bug]: No such file or directory '<my-installation-directory>/bullmq/commands/reprocessJob-6.lua'

### ISSUE THREAD

**Issue by sahil-springworks** [taskforcesh/bullmq#1952]
### Version

v0.4.2

### Platform

Python

### What happened?

Not able to initialize a Bull queue as shown in the screenshot below -> 

<img width="664" alt="Screenshot 2023-06-07 at 8 45 41 AM" src="https://github.com/taskforcesh/bullmq/assets/102500694/15d2f67d-f438-4670-9de0-b68291570001">

Getting the following response -> 
<img width="1456" alt="Screenshot 2023-06-07 at 8 45 48 AM" src="https://github.com/taskforcesh/bullmq/assets/102500694/2bb8e3fe-ceb9-46f4-8371-b5a451ba5f11">

System Preferences -
1. Using conda environment with python 3.10
2. Redis running on port 6379

Can someone please guide me if I'm initializing it wrongly? Thanks.

### How to reproduce.

<img width="664" alt="Screenshot 2023-06-07 at 8 45 41 AM" src="https://github.com/taskforcesh/bullmq/assets/102500694/15d2f67d-f438-4670-9de0-b68291570001">

### Relevant log output

```shell
ERROR:utils.logger:Error occurred in getting context: [Errno 2] No such file or directory: '/Users/srdevl216/anaconda3/envs/dobby-3.10/lib/python3.10/site-packages/bullmq/commands/reprocessJob-6.lua'
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by manast** [comment]
Could you please check with the latest version: https://pypi.org/project/bullmq/0.4.3/

**Comment by sahil-springworks** [comment]
Checked with v0.4.3 , getting this error ->

`ERROR:utils.logger:Error occurred in getting context: [Errno 2] No such file or directory: '/Users/srdevl216/anaconda3/envs/dobby-3.10/lib/python3.10/site-packages/bullmq/commands/changePriority-4.lua'`

**Comment by MRakox** [comment]
I solved it *temporarily* by downloading the scripts manually from [here](https://github.com/taskforcesh/bullmq/tree/master/src/commands); I think 3 or 4 of them were missing.

**Comment by manast** [comment]
They should be there. We are improving the CI to make the releases and hopefully after that these errors will go away.

**Comment by sahil-springworks** [comment]
Waiting eagerly. Thank you @manast

**Comment by roggervalf** [comment]
Hey @sahil-springworks please try version 0.5.0 https://github.com/taskforcesh/bullmq/releases/tag/vpy0.5.0

**Comment by sahil-springworks** [comment]
@roggervalf tried version 0.5.0 , getting this error ->

`ERROR:utils.logger:Error occurred in getting context: [Errno 2] No such file or directory: '/Users/srdevl216/anaconda3/envs/dobby-3.10/lib/python3.10/site-packages/bullmq/commands/addJob-8.lua'`

**Comment by roggervalf** [comment]
one extra fix was added into our release pipeline https://github.com/taskforcesh/bullmq/releases/tag/vpy0.5.1 please when you get some chance

**Comment by sahil-springworks** [comment]
@roggervalf yes, this particular issue was fixed in 0.5.1. Thanks a lot !
